### PR TITLE
chore: fix directory name in release please config

### PR DESCRIPTION
**Pull request by dyladan**
Directory name in release please config was incorrect.

Also sorted the config alphabetically.

### CODE REVIEW

File: release-please-config.json
Comment (dyladan) [inline release-please-config.json:20]: This is the fixed directory

File: release-please-config.json
Comment (dyladan) [inline release-please-config.json:37]: This was the incorrect directory

**Comment by dyladan** [comment]
@open-telemetry/javascript-approvers this is blocking the release

**Review by vmarchaud — APPROVED** [review]

**Pull request merged**
